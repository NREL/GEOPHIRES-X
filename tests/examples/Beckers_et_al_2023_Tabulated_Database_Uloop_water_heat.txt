#GEOPHIRES v3.0 Input File
#Last modified on 4/12/2023
#Example Description: Example 1 from "Tabulated Database of Closed-Loop Geothermal Systems Performance for Cloud-Based Technical and Economic Modeling of Heat Production and Electricity Generation"
#Koenraad Beckers Yaroslav Vasyliv Gabriela A. Bran-Anleu Mario Martinez Chad Augustine and Mark White
#Presented at the 48th Stanford Geothermal Workshop Stanford California February 6-8 2023
#This configuration is water filled U-loop with 1 lateral for direct-use heat usong water
#----------------------------------------------------------------------------------------
Is AGS, True
Closed-loop Configuration, 1,    ---uloop
End-Use Option, 2,        -----direct heat
Number of Production Wells, 1,      ---- 1 production well accounts for the full configuration - 1 up, one down, and a horizontal section
Number of Injection Wells, 0,		---- No injectors in a closed uloop
All-in Vertical Drilling Costs, 1000.0
All-in Nonvertical Drilling Costs, 1000.0
Exploration Capital Cost, 0
Production Flow Rate per Well, 20, ---- kg/s for water / 40 kg/s for sCO2
Cylindrical Reservoir Input Depth, 3.0, -----kilometers
Gradient 1, 60.0, ----deg.c/km
Total Nonvertical Length, 9000, ----- m
Production Well Diameter,8.5,		--- [inch]
Reservoir Depth, 3.0, -----kilometers
Injection Temperature, 60.0, -----deg.C
Plant Lifetime, 40, --- years
Ambient Temperature, 20, ----Deg.C "Dead-state temperature"
Electricity Rate, 0.10,     ----USD/kWh
Circulation Pump Efficiency, 0.8, -----[-]
#same as default Reservoir Thermal Conductivity, 3,     ---- W/m-K
#same as default Drilling Cost per Meter of Measured Depth, 1000,     ----$/m
#same as default Operation & Maintenance Cost of Surface Plant, 1.5,     -----%
#same as default Capital Cost for Surface Plant for Direct-use System, 100.00,    ----$/kWth
#same as default Dead-state Pressure, 1e5,     ---- bar

*** Economic/Financial Parameters ***
*************************************
Economic Model,4,			--- Should be 1 (FCR model) 2 (Standard LCOE/LCOH model) 3 (Bicycle model) 4 CLGS.
Fraction of Investment in Bonds,0.65,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflated Bond Interest Rate,0.07,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflated Equity Interest Rate,0.12,	--- [-] Required if Bicycle model is selected. See manual for details.
Inflation Rate,0.025,			--- [-] Required if Bicycle model is selected. See manual for details.
Combined Income Tax Rate,0.392,		--- [-] Required if Bicycle model is selected. See manual for details.
Gross Revenue Tax Rate,0,		--- [-] Required if Bicycle model is selected. See manual for details.
Reservoir Stimulation Capital Cost,0,				--- [M$/injection well] Reservoir stimulation capital cost per injection well

*** Simulation Parameters ***
Print Output to Console,1,		--- [-] Should be 0 (don't print results to console) or 1 (print results to console)
