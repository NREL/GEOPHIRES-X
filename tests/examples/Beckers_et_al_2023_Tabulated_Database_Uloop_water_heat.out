                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.6.0
 Simulation Date: 2024-10-15
 Simulation Time:  13:08
 Calculation Time:      0.856 sec


                           ***AGS/CLGS STYLE OUTPUT***

### Configuration ###
      End-Use: Direct-Use Heat
      Fluid: water
      Design: utube
      Flow rate:                                             20.0 kg/sec
      Lateral Length:                                      9000 meter
      Vertical Depth:                                      3 kilometer
      Geothermal Gradient:                                   60.0000 degC/km
      Wellbore Diameter:                                      8.5000 in
      Injection Temperature:                                 60.0 degC
      Thermal Conductivity:                                   3.00 W/m/K
 ### Reservoir Simulation Results ###
      Average Production Temperature:                       107.0 degC
      Average Production Pressure:                          207.0 bar
      Average Heat Production:                             3916.9 kW
      First Year Heat Production:                         43316.2 kWh
 ### Cost Results ###
      Total CAPEX:                                           15.5 MUSD
      Drilling Cost:                                         15.0 MUSD
      Surface Plant Cost:                                     0.5 MUSD
      OPEX:                                                  7.8 KUSD/yr
      LCOH:                                                 30.5 USD/MWh

                                        ******************************
                                        *  POWER GENERATION PROFILE  *
                                        ******************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET
             DRAWDOWN             TEMPERATURE             POWER              HEAT
                                   (deg C)                (MW)               (MW)
   0           1.0000                122.04               0.0000            4.6663
   1           0.9497                115.90               0.0000            4.6663
   2           0.9277                113.21               0.0000            4.4253
   3           0.9158                111.76               0.0000            4.2923
   4           0.9077                110.78               0.0000            4.2014
   5           0.9017                110.04               0.0000            4.1331
   6           0.8969                109.45               0.0000            4.0795
   7           0.8929                108.97               0.0000            4.0352
   8           0.8895                108.56               0.0000            3.9975
   9           0.8866                108.20               0.0000            3.9650
  10           0.8840                107.88               0.0000            3.9363
  11           0.8817                107.60               0.0000            3.9107
  12           0.8796                107.35               0.0000            3.8877
  13           0.8777                107.11               0.0000            3.8667
  14           0.8760                106.90               0.0000            3.8475
  15           0.8744                106.71               0.0000            3.8298
  16           0.8729                106.52               0.0000            3.8134
  17           0.8715                106.35               0.0000            3.7981
  18           0.8702                106.19               0.0000            3.7838
  19           0.8689                106.04               0.0000            3.7704
  20           0.8678                105.90               0.0000            3.7577
  21           0.8667                105.77               0.0000            3.7458
  22           0.8656                105.64               0.0000            3.7345
  23           0.8647                105.52               0.0000            3.7237
  24           0.8637                105.41               0.0000            3.7135
  25           0.8628                105.30               0.0000            3.7038
  26           0.8620                105.19               0.0000            3.6944
  27           0.8611                105.09               0.0000            3.6855
  28           0.8604                105.00               0.0000            3.6769
  29           0.8596                104.90               0.0000            3.6687
  30           0.8589                104.81               0.0000            3.6608
  31           0.8582                104.73               0.0000            3.6531
  32           0.8575                104.65               0.0000            3.6458
  33           0.8568                104.57               0.0000            3.6387
  34           0.8562                104.49               0.0000            3.6318
  35           0.8556                104.41               0.0000            3.6252
  36           0.8550                104.34               0.0000            3.6187
  37           0.8544                104.27               0.0000            3.6125
  38           0.8539                104.20               0.0000            3.6065

                              ***************************************************************
                              *  HEAT AND/OR ELECTRICITY EXTRACTION AND GENERATION PROFILE  *
                              ***************************************************************
  YEAR               HEAT                       HEAT                RESERVOIR           CUM PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                  35.1                        39.0                  3.65                   3.71
   2                  32.2                        35.8                  3.52                   7.10
   3                  31.1                        34.5                  3.40                  10.38
   4                  30.3                        33.7                  3.28                  13.58
   5                  29.8                        33.2                  3.16                  16.73
   6                  29.4                        32.7                  3.04                  19.84
   7                  29.1                        32.4                  2.92                  22.91
   8                  28.9                        32.1                  2.81                  25.95
   9                  28.6                        31.8                  2.69                  28.97
  10                  28.4                        31.6                  2.58                  31.97
  11                  28.3                        31.4                  2.47                  34.96
  12                  28.1                        31.2                  2.35                  37.92
  13                  28.0                        31.1                  2.24                  40.87
  14                  27.8                        30.9                  2.13                  43.80
  15                  27.7                        30.8                  2.02                  46.73
  16                  27.6                        30.7                  1.91                  49.64
  17                  27.5                        30.5                  1.80                  52.54
  18                  27.4                        30.4                  1.69                  55.42
  19                  27.3                        30.3                  1.58                  58.30
  20                  27.2                        30.2                  1.47                  61.17
  21                  27.1                        30.1                  1.36                  64.03
  22                  27.0                        30.0                  1.26                  66.89
  23                  27.0                        30.0                  1.15                  69.73
  24                  26.9                        29.9                  1.04                  72.57
  25                  26.8                        29.8                  0.93                  75.40
  26                  26.8                        29.7                  0.83                  78.23
  27                  26.7                        29.7                  0.72                  81.04
  28                  26.7                        29.6                  0.61                  83.86
  29                  26.6                        29.5                  0.51                  86.66
  30                  26.5                        29.5                  0.40                  89.46
  31                  26.5                        29.4                  0.29                  92.26
  32                  26.4                        29.4                  0.19                  95.05
  33                  26.4                        29.3                  0.08                  97.83
  34                  26.3                        29.3                 -0.02                 100.61
  35                  26.3                        29.2                 -0.13                 103.38
  36                  26.3                        29.2                 -0.23                 106.15
  37                  26.2                        29.1                 -0.34                 108.92
  38                  26.2                        29.1                 -0.44                 111.68
  39                  26.1                        29.0                 -0.55                 114.44
